src/components/ChainSelectionPopover/ChainSelectionPopover.tsx

```diff
@@ -12,7 +12,7 @@ import {
   NetworkButton,
   NetworkIcon,
 } from "./ChainSelectionPopover.styles";
-import addAndSwitchToEthereumChain from "./helpers/addAndSwitchToEthereumChain";
+import addAndSwitchToChain from "./helpers/addAndSwitchToChain";
```

Comment:
the file name is addAndSwitchToDefaultChain.ts

I'm getting an error `Module not found: Error: Can't resolve './helpers/addAndSwitchToChain' in '/Users/anon/Development/airswap-web/src/components/ChainSelectionPopover'`